dftimewolf/lib/processors/turbinia_base.py

```diff
@@ -99,6 +101,23 @@ def __init__(
     self.turbinia_api = str()
     os.environ['GRPC_POLL_STRATEGY'] = 'poll'
 
+  def _decode_api_response(self, data: Any) -> Any:
+    """Decodes ApiResponse data into a Python object."""
+    if not isinstance(data, ApiResponse):
+      return data
+    data_attribute = None
+    response = ''
+    try:
+      if data_attribute := getattr(data, 'data'):
+        response = data_attribute
+      if not data_attribute:
+        if data_attribute := getattr(data, 'raw_data'):
+          response = json.loads(data_attribute)
+    except (
+      AttributeError, json.JSONDecodeError) as exception:
```

Comment:
Minor nit, but the whitespace might be off here:  I think it's supposed to be indented 4 spaces for the line continuation.

Reply:
Looks like that's the case for a few other line continuation indents later in the file as well.